src/test/regress/sql/with_clause.sql

```diff
@@ -405,3 +405,21 @@ WITH e AS (
 ) SELECT * FROM r JOIN h USING (a) JOIN h i USING (a);
 DROP TABLE d;
 DROP TABLE r;
+
+-- Greenplum fails to execute SELECT INTO and CREATE TABLE AS statements, whose
+-- queries contain modifying CTEs, because Greenplum cannot have two writer
+-- segworker groups, and during execution an error is thrown. Showing
+-- the error during planning stage would be more effective, therefore this test
+-- checks this behaviour.
+--start_ignore
+drop table if exists with_dml;
+drop table if exists t_new;
+--end_ignore
+create table with_dml (i int, j int) distributed by (i);
+with cte as
```

Comment:
If the fix was for planning, then maybe it’s not worth doing the execution in the tests?

Reply:
Changed the test